src/main/resources/META-INF/services/org.apache.pirk.responder.wideskies.spi.ResponderPlugin

```diff
@@ -0,0 +1,5 @@
+org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponder
```

Comment:
No - every option requires user specification. I had originally thought that we would have the user specify the interface implementation (via class name) via the command line/prop file and then instantiate through reflection. 

I like the service-plugin model and am in favor of proceeding with it. It just wasn't what I thought we were doing when we started this PR (recall that we started with a ResponderLauncher interface + user specification of implementation) and wanted to make sure that we were clear on the tradespace.